tests/unit_tests/tools/skill_tools/test_skill_cli.py

```
@@ -0,0 +1,388 @@
+        args.skill_args = skill_args or []
+        args.marketplace = None
+        args.email = None
+        args.password = None
+        args.owner = ""
+        for k, v in kwargs.items():
+            setattr(args, k, v)
+        return args
+
+    @patch("datus.cli.skill_cli._get_manager")
+    @patch("datus.cli.skill_cli._cmd_list")
+    def test_dispatch_list(self, mock_cmd, mock_mgr):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_mgr`

(ARG002)